docs/en-US/config.yml.md

```diff
@@ -76,6 +76,10 @@ hosts:
   - one.wordpress.test
 ```
 
+### php
+
+This option is similar to the previous one, the difference is that let you to set the PHP vverson to use during the provisiong of that site.
```

Comment:
```suggestion
This option is similar to the previous one, the difference is that it lets you to set the PHP version to use during the provisioning of that site.
```

Reply:
note that this isn't actually true, this is meant to set both the PHP version during provisioning _**and**_ when it runs in the browser. It's intended as a complete replacement of `nginx_upstream`.

IMO this also shouldn't be added until it reaches the `stable` branch, and it needs the version it's available from listed.